docs/api/vulnerability-info.md

```diff
@@ -33,6 +33,16 @@ The data file schema does not allow for modification or redaction of known vulne
 Therefore if a server's vulnerability data source (for example the [GitHub Advisories Database](https://github.com/advisories)) modifies an existing advisory, the NuGet server must modify the page that the vulnerability information was previously reported.
 One way to achieve this with the suggested partition scheme is to treat all vulnerability modifications and deletions as a trigger to regenerate the complete `base.json` file, and empty `updates.json`.
 
+## Reusing nuget.org's vulnerability data
```

Comment:
I can't decide whether this content should be in the server implementation guide document, rather than here.

Any comments?

Reply:
I think it should be in the implementation guide as a specific example. To me it seems like the point generalizes to many potential upstreaming modes (not just for vuln data).